doc/user/content/ingest-data/postgres-amazon-rds.md

```diff
@@ -0,0 +1,472 @@
+
+    Set **Parameter group family** to your PostgreSQL version and **Type** to **DB Parameter Group**.
+
+1. Edit the new parameter group and set the `rds.logical_replication` parameter to `1`.
+
+1. [Associate the RDS parameter group to your database](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_WorkingWithDBInstanceParamGroups.html#USER_WorkingWithParamGroups.Associating).
+
+    Use the **Apply Immediately** option. The database must be rebooted in order for the parameter group association to take effect. Keep in mind that rebooting the RDS instance can affect database performance.
+
+    Do not move on to the next step until the database **Status** is **Available** in the RDS Console.
+
+1. Back in your `psql` shell, verify that replication is now enabled:
+
+    ``` sql
+    SELECT name, setting
+      FROM pg_settings
+      WHERE name = 'rds.logical_replication';
+    ```
+    <p></p>
+
+    ``` nofmt
+            name             | setting
+    -------------------------+---------
+    rds.logical_replication  | on
+    (1 row)
+    ```
+
+    If replication is still not enabled, [reboot the database](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_RebootInstance.html).
+
+## Step 2. Create a publication
+
+Once logical replication is enabled, the next step is to create a publication with the tables that you want to replicate to Materialize. You'll also need a user for Materialize with sufficient privileges to manage replication.
+
+1. For each table that you want to replicate to Materialize, set the [replica identity](https://www.postgresql.org/docs/current/sql-altertable.html#SQL-ALTERTABLE-REPLICA-IDENTITY) to `FULL`:
+
+    ```sql
+    ALTER TABLE <table1> REPLICA IDENTITY FULL;
+    ```
+
+    ```sql
+    ALTER TABLE <table2> REPLICA IDENTITY FULL;
+    ```
+
+    `REPLICA IDENTITY FULL` ensures that the replication stream includes the previous data of changed rows, in the case of `UPDATE` and `DELETE` operations. This setting enables Materialize to ingest PostgreSQL data with minimal in-memory state. However, you should expect increased disk usage  in your PostgreSQL database.
+
+1. Create a [publication](https://www.postgresql.org/docs/current/logical-replication-publication.html) with the tables you want to replicate:
+
+    _For specific tables:_
+
+    ```sql
+    CREATE PUBLICATION mz_source FOR TABLE <table1>, <table2>;
+    ```
+
+    _For all tables in the database:_
+
+    ```sql
+    CREATE PUBLICATION mz_source FOR ALL TABLES;
+    ```
+
+    The `mz_source` publication will contain the set of change events generated from the specified tables, and will later be used to ingest the replication stream.
+
+    Be sure to include only the tables you need. If the publication includes additional tables, Materialize will waste resources on ingesting and then immediately discarding the data.
```

Comment:
I think it's worth to call out that the database may be impacted as well. If there is a replication log that is not consumed from, the log will grow indefinitely until it runs out of disk space.

Reply:
These messages are consumed in Materialize, just ignored; we won't cause the machine to run out of disk space (bugs and unforeseen outages notwithstanding).

Reply:
Oh, I meant the database is eventually running out of space. My terminology might be slightly off, but if the replication log is not actively consumed by anyone (there is a publication but no consumer or stalled consumer), the database log will just grow indefinitely until the database runs out of disk space.

It was more meant as a warning: if you aren't careful here, you might cause trouble for the upstream database (eg, if you just drop the source, the database might eventually fall over).

Reply:
@sthm, what would cause the replication log to grow indefinitely? Might that happen if Materialize's connection fails (e.g., changed RDS IP address) or if the source cluster in Materialize crashes for some reason?